src/util.h

```diff
@@ -364,6 +364,11 @@ inline v8::Local<v8::String> FIXED_ONE_BYTE_STRING(v8::Isolate* isolate,
   return OneByteString(isolate, data, N - 1, v8::NewStringType::kInternalized);
 }
 
+inline v8::MaybeLocal<v8::String> Uint32ToString(v8::Local<v8::Context> context,
+                                                 uint32_t value) {
+  return v8::Uint32::New(v8::Isolate::GetCurrent(), value)->ToString(context);
```

Comment:
This V8 operation should not be failable: https://github.com/nodejs/node/pull/60846